apps/src/code-studio/pd/workshop_dashboard/components/workshop_enrollment.jsx

```diff
@@ -81,77 +41,33 @@ const WorkshopEnrollment = React.createClass({
     }
 
     const sortedEnrollments = _.sortBy(this.props.enrollments, ['last_name', 'first_name']);
-    const enrollmentRows = sortedEnrollments.map((enrollment, i) => {
-      let deleteCell;
-      if (enrollment.attended) {
-        // Don't give the option to delete an enrollment once the teacher has been marked attended.
-        deleteCell = <td />;
-      } else {
-        deleteCell = (
-          <td
-            style={styles.clickTarget}
-            onClick={this.handleClickDelete}
-            data-id={enrollment.id}
-            data-first_name={enrollment.first_name}
-            data-last_name={enrollment.last_name}
-            data-email={enrollment.email}
-          >
-            <i className="fa fa-minus" />
-          </td>
-        );
-      }
 
+    if (this.shouldShowPreSurveys()) {
       return (
-        <tr key={i}>
-          {deleteCell}
-          <td>{i + 1}</td>
-          <td>{enrollment.first_name}</td>
-          <td>{enrollment.last_name}</td>
-          <td>{enrollment.email}</td>
-          <td>{enrollment.district_name}</td>
-          <td>{enrollment.school}</td>
-          {this.props.accountRequiredForAttendance && <td>{enrollment.user_id ? 'Yes' : 'No'}</td>}
-        </tr>
+        <Tabs activeKey={this.props.activeTab} onSelect={this.props.onTabSelect} id="enrollment-tabs">
+          <Tab eventKey={0} title="Attendee School Info">
+            <WorkshopEnrollmentSchoolInfo
```

Comment:
nit - you could probably assign this element to a const before the if statement and then reuse it in both clauses